src/common/congestionCharges.ts

```diff
@@ -0,0 +1,36 @@
+import { logErrorReturnLogId } from "@/logger/logger";
+import supabase from "@/supabaseClient";
+
+export type CongestionChargeDetails = {
+    postcode: string;
+    congestionCharge: boolean;
+};
+
+export type CongestionChargeError = "failedToRetrieveCongestionChargeDetails";
+
+export type CongestionChargeReturnType =
+    | {
+          data: CongestionChargeDetails[];
+          error: null;
+      }
+    | {
+          data: null;
+          error: { type: CongestionChargeError; logId: string };
+      };
+
+export const checkForCongestionCharge = async (
+    postcodes: (string | null)[]
+): Promise<CongestionChargeReturnType> => {
```

Comment:
Maybe a line break here for slightly better readability?

Reply:
If I do this then lint_fix changes it back - I guess it doesn't like the first line to be empty?

Reply:
I guess the linter doesn't agree with my eyes 😞 yeah just leave it in that case